### DISCUSSION TITLE
Default Terminal is now supported on Windows 10!

### DISCUSSION THREAD

**Discussion by DHowett** [microsoft/terminal#15505] [Announcements]
_You can finally set Windows Terminal as the default host for console applications on Windows 10!_ 🎉

After you install [KB5026435](https://support.microsoft.com/en-us/topic/may-23-2023-kb5026435-os-build-19045-3031-preview-2751b693-5544-4110-bc0c-feb8dd7336b3), you will be able to access the Default Terminal settings in the console host and Developer settings page.

<img width="285" alt="image" src="https://github.com/microsoft/terminal/assets/189190/f1ef0d07-be4d-4a32-b393-b23528a46d99">

<img width="608" alt="image" src="https://github.com/microsoft/terminal/assets/189190/ae24a502-cfd1-412a-a550-702ae7db2bf0">

After you install Windows Terminal v1.17, you will also be able to access the setting inside Terminal. _Default terminal is supported in earlier versions,_ but there is no code in those versions to detect that Windows 10 supports it... so it won't give you the option.

<img width="876" alt="image" src="https://github.com/microsoft/terminal/assets/189190/d908ad6f-a021-4542-a652-aa611b93e84c">

**Comment by wuyilingwei** [comment]
wow

**Comment by PRO-2684** [comment]
🥳

**Comment by LucasOe** [comment]
Are third party terminals like Alacritty or ConEmu also supported? That would be amazing!

**Reply by j4james** [reply]
I believe it's theoretically possible, but there is a certain amount of work those terminals would need to do to support this. I know a number of them already have open issues for it though.

* [Contour](https://github.com/contour-terminal/contour/issues/605)
* [WezTerm](https://github.com/wez/wezterm/issues/2416)
* [Alacritty](https://github.com/alacritty/alacritty/issues/6036)

**Comment by jamespack** [comment]
SO PUMPED for this for my work life. 🙌
![SoExcited~GIF](https://github.com/microsoft/terminal/assets/2086722/e4b440ab-c151-46da-ab32-eeecdc2e50a6)

**Reply by Tanner-hue** [reply]
😂😂😂😂😂😂😂😂😂😂😂😂😂😂😂

**Comment by ZHJ0125** [comment]
Win10 YYDS

**Reply by whwdzg** [reply]
说得好

**Comment by ianjoneill** [comment]
In the settings app, the text above the drop-down list-box is cut-off for me. This includes at full-screen resolution (1366x768) on my old laptop.

![image](https://github.com/microsoft/terminal/assets/5821575/0af8d793-7a11-42af-ad01-c75658dd3f7b)

Also, why does the linked KB article not mention this change at all? You'd think it would be a highlight!

**Reply by KalleOlaviNiemitalo** [reply]
The text is cut in Finnish as well.  Filed <https://aka.ms/AAlffuv>.

**Comment by Tanner-hue** [comment]
There is one issue with the app @DHowett so the command shell profiles just don't save so like if I made a Command shell profile it would save but then when I close the app then open it again and see if it saved I just don't see it being saved

**Reply by zadjii-msft** [reply]
Can you file a new issue for this? This doesn't sound like something that should have regressed just because defterm is now supported on Windows 10....

Thanks!

**Reply by Tanner-hue** [reply]
> Can you file a new issue for this? This doesn't sound like something that should have regressed just because defterm is now supported on Windows 10....
> 
> Thanks!

Ok but I now solved it
